src/main/kotlin/ar/edu/unahur/obj2/semillasAlViento/Planta.kt

```diff
@@ -1,35 +1,36 @@
 package ar.edu.unahur.obj2.semillasAlViento
 
 abstract class Planta(val anioObtencionSemilla: Int, var altura: Float) {
+//  Simplicidad: la altura no debería poder cambiarse, el enunciado lo aclara.
   fun esFuerte() = this.horasDeSolQueTolera() > 10
 
+//  (Des)acoplamiento: Este método debería pertenecer a la clase "Parcela".
   fun parcelaTieneComplicaciones(parcela: Parcela) =
     parcela.plantas.any { it.horasDeSolQueTolera() < parcela.horasSolPorDia }
 
   abstract fun horasDeSolQueTolera(): Int
+
   abstract fun daSemillas(): Boolean
```

Comment:
Una solución posible es hacer un `template method` en este caso la condioción que se repite es si `esFuerte()`.
```suggestion
  open fun daSemillas() = this.esFuerte()
  
  // Y después podrían hacer en las subclases
  override fun daSemillas() = super.daSemillas() || altura > 0.4
```